recipes/webargs/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "webargs" %}
+{% set version = "1.4.0" %}
+{% set sha256 = "6ec4a2defc02819d9a7a6c8bc304e1d3a15fea0aff570c4b094f44fe2852a6c3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - marshmallow >=2.0.0
```

Comment:
Please drop anything from `build` that is not strictly a requirement at build time. Probably just need `python` and `setuptools`.